rpcserver.go

```diff
@@ -1063,11 +1063,10 @@ func (r *rpcServer) ListPeers(ctx context.Context,
 	return resp, nil
 }
 
-// WalletBalance returns the sum of all confirmed unspent outputs under control
+// WalletBalance returns total unspent outputs(confirmed or unconfirmed), all
+// confirmed unspent outputs and all unconfirmed unspent outputs under control
 // by the wallet. This method can be modified by having the request specify
 // only witness outputs should be factored into the final output sum.
-// TODO(roasbeef): split into total and confirmed/unconfirmed
-// TODO(roasbeef): add async hooks into wallet balance changes
```

Comment:
I think this TODO needs to be added back - it is not implemented or in scope for this PR.